ui/package.json

```diff
@@ -24,6 +24,7 @@
     "@babel/polyfill": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
+    "@fortawesome/fontawesome-free": "^5.9.0",
```

Comment:
🏰 ? :)

Reply:
The organization on the Font Awesome GitHub project is in fact called Fort Awesome. That's where the name comes from with just a single letter variation: Font Awesome.

One can read more about the name here: https://www.impresspages.org/awesome-q-a-with-font-awesome-developer-dave-gandy#.XSWsT1UzZhE